pyqtgraph/graphicsItems/PlotItem/PlotItem.py

```diff
@@ -763,9 +763,8 @@ def writeSvgCurves(self, fileName=None):
 
             for item in self.curves:
                 if isinstance(item, PlotCurveItem):
-                    color = fn.colorStr(item.pen.color())
-                    opacity = item.pen.color().alpha() / 255.
-                    color = color[:6]
+                    color = item.pen.color()
+                    hrrggbb, opacity = color.name(), color.alphaF()
```

Comment:
Out of curiosity, what does the `h` in `hrrggbb` stand for?

Reply:
hex or hash. The name() method returns a str of the form '#rrggbb'. It's to remind that a hex or hash has already been prepended.